eclipse/plugin.xml

```diff
@@ -138,7 +139,8 @@
             highlightPreferenceKey="saros.annotations.viewport.highlight.default"
             highlightPreferenceValue="false"
             includeOnPreferencePage="true"
-            label="DPP Default Viewport"
+            label="Saros Default User Viewport"
+            icon="icons/obj16/contact_saros_obj.png"
```

Comment:
Is there a reason to use this icon? How about using the Saros icon instead to make the options easier to find?

Reply:
![image](https://user-images.githubusercontent.com/41400679/62955418-50178f00-bdf1-11e9-9b85-358f289abae9.png)

It is this icon, do you prefer the Saros Icon instead of the Saros User Icon ?

Reply:
For me, the Saros User icon just reminds me of the old MSN contact icon. I don't particularly associate it with Saros.

I would use the Saros Icon, but if you think that the Sarus User Icon works as well/better, I don't mind using it instead.

Reply:
Ok changed to Saros Icon.